devtools/packaging/scripts/android/config.proto

```diff
@@ -0,0 +1,197 @@
+syntax = "proto3";
```

Comment:
Where does this file come from? What is its license?

Reply:
I took it from: https://github.com/android/app-bundle-samples/blob/main/PlayAssetDelivery/BundletoolScriptSample/config.proto

However, it seems that config.proto in above repo is from the [bundletool repository](https://github.com/google/bundletool/blob/master/src/main/proto/config.proto).
 
The license of both the source code is Apache 2.0.